linetools/isgm/utils.py

```diff
@@ -360,29 +347,61 @@ def table_from_complist(complist):
     Special columns: 'name', 'comment', 'logN', 'sig_logN', 'flag_logN'
     See also complist_from_table()
     """
+    key_order = ['RA', 'DEC', 'name', 'z_comp', 'sig_z', 'Z', 'ion', 'Ej',
+                 'vmin', 'vmax','ion_name', 'flag_N', 'logN', 'sig_logN',
+                 'b','sig_b', 'specfile']
```

Comment:
`reliability` should be included

Reply:
It is introduced later.

Reply:
Fine. Please update the docstring accordingly.

Reply:
added